apps/examples/testcase/le_tc/network/tc_net_pbuf.c

```diff
@@ -0,0 +1,210 @@
+* @precondition
+* @postcondition
+*/
+static void tc_net_pbuf_layer_transport_p(int fd)
+{
+	int ret = 0;
+	struct pbuf_test_args args;
+
+	args.layer = PBUF_TRANSPORT;
+	args.len = PBUF_TRANSPORT_HLEN;
+	args.type = PBUF_RAM;
+
+	ret = ioctl(fd, TESTIOC_NET_PBUF, &args);
+	TC_ASSERT_GEQ("pbuf", ret, 1);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase             tc_net_pbuf_layer_ip_p
+* @brief		check for the memory allocation for the pbuf by ip layer
+* @scenario
+* @apicovered           pbuf_alloc()
+* @precondition
+* @postcondition
+*/
+static void tc_net_pbuf_layer_ip_p(int fd)
+{
+	int ret = 0;
+	struct pbuf_test_args args;
+
+	args.layer = PBUF_IP;
+	args.len = PBUF_IP_HLEN;
+	args.type = PBUF_RAM;
+
+	ret = ioctl(fd, TESTIOC_NET_PBUF, &args);
+	TC_ASSERT_GEQ("pbuf", ret, 1);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase             tc_net_pbuf_layer_link_p
+* @brief		check for the memory allocation for the pbuf by link layer
+* @scenario
+* @apicovered           pbuf_alloc()
+* @precondition
+* @postcondition
+*/
+static void tc_net_pbuf_layer_link_p(int fd)
+{
+	int ret = 0;
+	struct pbuf_test_args args;
+
+	args.layer = PBUF_LINK;
+	args.len = PBUF_LINK_HLEN;
+	args.type = PBUF_RAM;
+
+	ret = ioctl(fd, TESTIOC_NET_PBUF, &args);
+	TC_ASSERT_GEQ("pbuf", ret, 1);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase             tc_net_pbuf_layer_raw_tx_p
+* @brief		check for the memory allocation for the pbuf by encapsulating link layer headers (e.g. 802.11)
+* @scenario
+* @apicovered           pbuf_alloc()
+* @precondition
+* @postcondition
+*/
+static void tc_net_pbuf_layer_raw_tx_p(int fd)
+{
+	int ret = 0;
+	struct pbuf_test_args args;
+
+	args.layer = PBUF_RAW_TX;
+	args.len = PBUF_LINK_ENCAPSULATION_HLEN;
+	args.type = PBUF_RAM;
+
+	ret = ioctl(fd, TESTIOC_NET_PBUF, &args);
+	TC_ASSERT_GEQ("pbuf",ret, 1);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase             tc_net_pbuf_layer_raw_p
+* @brief		check for the memory allocation for the pbuf by RX buffers
+* @scenario
+* @apicovered           pbuf_alloc()
+* @precondition
+* @postcondition
+*/
+static void tc_net_pbuf_layer_raw_p(int fd)
+{
+	int ret = 0;
+	struct pbuf_test_args args;
+
+	args.layer = PBUF_RAW;
+	args.len = 0;
+	args.type = PBUF_RAM;
+
+	ret = ioctl(fd, TESTIOC_NET_PBUF, &args);
+	TC_ASSERT_GEQ("pbuf", ret, 1);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase             tc_net_pbuf_layer_n
+* @brief		passed incorrect layer
+* @scenario
+* @apicovered           pbuf_alloc()
+* @precondition
+* @postcondition
+*/
+static void tc_net_pbuf_layer_n(int fd)
+{
+	int ret = 0;
+	struct pbuf_test_args args;
+
+	args.layer = PBUF_TEST;
+	args.len = 0;
+	args.type = PBUF_RAM;
+
+	ret = ioctl(fd, TESTIOC_NET_PBUF, &args);
+	TC_ASSERT_EQ("pbuf", ret, 0);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase             tc_net_pbuf_type_n
+* @brief		passed incorrect type. type parameter decides how and where the pbuf
+*                       should be allocated.
```

Comment:
Could you stretch this comment in one line or wrap the comment in line 109 too for similarity?
`* @brief		    check for the memory allocation for the pbuf by encapsulating link layer headers (e.g. 802.11)`

Reply:
done